src/styles/abstracts/_variables.scss

```diff
@@ -1,7 +1,5 @@
 $border-height-sm: 0.0625rem;
 
-$header-font: 1.875rem;
-
 $breakpoint-sm: 640px;
 $breakpoint-md: 1024px;
```

Comment:
Please use rem.

Reply:
I think it's better to use `em` or  `px` in media queries. These variables are used as media query breakpoints
here are some references:
- https://medium.com/@barrypeng6/why-should-not-use-rem-unit-in-media-query-5645d0163ce5#:~:text=Unit%20rem%20in%20media%20queries,queries%20before%20the%20bug's%20fixed.
- https://zellwk.com/blog/media-query-units/
 
Thanks